scripts/python/lib/MojGithub.py

```diff
@@ -0,0 +1,91 @@
+from functools import reduce
+from github import Github
+from github import PaginatedList
+from github import Repository
+
+# This class is an interface for interacting with a GitHub org and its repos at an admin level, it can be pointed at any org
+# But please be aware it has been written with the ministryofjustice org in mind
+
+class MojGithub:
+    def __init__(self, org: str, org_token: str) -> None:
+        self.git = Github(org_token)
+        self.org = self.git.get_organization(org)
+        self.__repos = None     
+        self.__public_repos = None
+        self.__private_repos = None
+      
+    @property 
+    def repos(self) -> list:
+        """Returns all repos for given organization
+        This does include archived repos.
+        This does not include forks repos.
+        
+        Please see the function get_unarchived_repos to ommit archived repos
+        """
+        if self.__repos == None:
+            self.__repos = self._get_repo_type(type="all")
+        return self.__repos
+       
+    @property
+    def public_repos(self) -> list:
+        """Returns all public repos for given organization
+        This does include archived repos.
+        This does not include forks repos.
+        
+        Please see the function get_unarchived_repos to ommit archived repos
+        """
+        if self.__public_repos == None:
+            self.__public_repos = self._get_repo_type(type="public")
+        return self.__public_repos
+    
+    @property
+    def private_repos(self) -> list:
+        """Returns all private repos for given organization.\n
```

Comment:
Is the \n needed at the end of the line within the comment, if not and it can be deleted there are multiple lines below that have it as well

Reply:
Yep needed